torchinductor/ir.py

```diff
@@ -1481,6 +1481,12 @@ def is_stride_ordered(self, order):
                 return False
         return True
 
+    def is_channels_last_stride_ordered(self):
+        # create channels_last order(NCHW, NCDHW, the C is the first order).
```

Comment:
What's the behavior of `conv1d`?

Reply:
For conv1d case, aten.convolution always returns a continuous tensor even the input is NHC format.